packages/runtime/src/runtimes/react/controls/style.ts

```diff
@@ -63,6 +65,14 @@ function useStyleControlCssObject(
           borderBottomRightRadius: borderRadiusToString(borderRadius?.borderBottomRightRadius) ?? 0,
           borderBottomLeftRadius: borderRadiusToString(borderRadius?.borderBottomLeftRadius) ?? 0,
         }),
+        ...(properties.includes(StyleControlProperty.TextStyle) && {
+          ...(textStyle?.fontFamily && { fontFamily: textStyle.fontFamily }),
+          ...(textStyle?.letterSpacing && { letterSpacing: textStyle.letterSpacing }),
+          ...(textStyle?.fontSize && { fontSize: fontSizeToString(textStyle.fontSize) ?? '16px' }),
```

Comment:
Since we're already checking if `fontSize` is falsy here, why not just make `fontSizeToString` always return a string? The argument can be `NonNullable<FontSizePropertyData>`. This way there's not an arbitrary 16px value that's never used.

Reply:
Yeah true I updated it.